src/model_code/budget_equation.py

```diff
@@ -70,18 +70,22 @@ def create_savings_grid():
     return savings_grid
 
 
-def calc_net_income_pensions(gross_income):
-    ssc = calc_health_ltc_contr(gross_income)
-    inc_tax = calc_inc_tax(gross_income - ssc)
-    net_income = gross_income - inc_tax - ssc
-    return net_income
+def calc_net_income_pensions(gross_income, options):
+    gross_income_full = gross_income * options["wealth_unit"]
```

Comment:
I increased the income by the wealth unit and computed taxes. Then I converted back.